data/migrations/20190116144803_users_table.js

```diff
@@ -0,0 +1,18 @@
+
+exports.up = function (knex) {
+    return knex.schema.createTable('users', table => {
+        table
```

Comment:
Good job setting up a schema here. I would recommend a different styling here because this can get tedious to read if there were more tables.